bindep.txt

```diff
@@ -1,6 +1,10 @@
-libxml2-devel [platform:rpm]
+openssl-devel [platform:rpm compile]
+gcc [platform:rpm compile]
+libcurl-devel [platform:rpm compile]
+libxml2-devel [platform:rpm compile]
 python3-pycurl [platform:rpm]
 python3-netaddr [platform:rpm]
 python3-jmespath [platform:rpm]
-python3-passlib [platform:rpm]
+python3-passlib [platform:fedora]
```

Comment:
Please be aware that oVirt doesn't support Fedora releases, we support only CentOS 8 or CentOS Stream

Reply:
ack. I'll delete this.